README.md

```diff
@@ -12,6 +12,10 @@ There are two npm packages for easier use:
 |  [airship-components](https://www.npmjs.com/package/@carto/airship-components) | ![status](https://img.shields.io/badge/status-in%20progress-orange.svg) | [![npm version](https://badge.fury.io/js/%40carto%2Fairship-components.svg)](https://badge.fury.io/js/%40carto%2Fairship-components) | Web components for Location Intelligence apps. |
 |  [airship-icons](https://www.npmjs.com/package/@carto/airship-icons) | ![status](https://img.shields.io/badge/status-in%20progress-orange.svg) | [![npm version](https://badge.fury.io/js/%40carto%2Fairship-icons.svg)](https://badge.fury.io/js/%40carto%2Fairship-icons) | High quality icons set |
 
+## Documentation
+
+The library is documented with [Catalog](https://www.catalog.style), check out our [documentation website](https://carto-airship.netlify.com/#/)
```

Comment:
Is it necessary to say what we use to build the documentation?

Reply:
No, it's just that it felt too short without it.

I'll remove the catalog bit and just leave the Check out our blabla

We should ask ourselves if we want to have some short documentation here as well, as you mentioned.